learntools/machine_learning/nbs/tut3.ipynb

```diff
@@ -0,0 +1,261 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "dc1e7b02-f022-48ca-a270-21f4b0c70835",
+    "_uuid": "0696f41450f17b2e0ebbbbb30a29053cde7de211"
+   },
+   "source": [
+    "**[Machine Learning Course Home Page](kaggle.com/learn/machine-learning).**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "# Selecting and Data for Modeling\n",
```

Comment:
Title feels a bit awkward. At first glance, it seems like a typo for "Selecting Data for Modeling" (maybe it is?)

Reply:
Yep. Typo